### ISSUE TITLE
Doesn't trim space after open brackets | C

### ISSUE THREAD

**Issue by shakram02** [cs50/style50#61]
Thanks for providing such a cool tool in CS50, it's pretty helpful!

I'm not sure if it's a bug but I guess it might be nice to add

Input
```C
int main( void)
{
}
```

Output
```
Looks good!
But consider adding more comments!
```

Expected Output (trims space before `void`)
```C
int main(void)
{
}
And consider adding more comments!
```

Same happens in `for` loops if I type `for( int i = 0; i < n; i++)`

**Comment by dmalan** [comment]
Hm, a good catch. Let me check with @crossroads1112 as to whether the underlying tool, `astyle`, can catch this!

**Comment by shakram02** [comment]
We need this flag I guess, I'm quoting the [docs](http://astyle.sourceforge.net/astyle.html)
```
--unpad-paren / -U 
```
Remove extra space padding around parens on the inside and outside. Any end of line comments will remain in the original column, if possible. This option can be used in combination with the paren padding options pad‑paren, pad‑paren‑out, pad‑paren‑in, and pad‑header above. Only padding that has not been requested by other options will be removed.

For example, if a source has parens padded on both the inside and outside, and you want inside only. You need to use unpad-paren to remove the outside padding, and pad‑paren‑in to retain the inside padding. Using only pad‑paren‑in> would not remove the outside padding.

```C
if ( isFoo( ( a+2 ), b ) )
    bar ( a, b );

// becomes (with no padding option requested):

if(isFoo((a+2), b))
    bar(a, b);
```

I'll try to run the code locally and see what happens
